docs/guides/dynamo_run.md

```diff
@@ -447,7 +442,7 @@ The output looks like this:
 
 ### Defaults
 
-The input defaults to `in=text`. The output will default to `mistralrs` engine. If not available whatever engine you have compiled in (so depending on `--features`).
+The input defaults to `in=text`. The output will default to `mistralrs` engine, unless it is disabled with `--no-default-features` in which case vllm is used.
```

Comment:
`out=mistralrs` for symmetry with `in=text`
```suggestion
The input defaults to `in=text`. The output will default to `out=mistralrs` engine, unless it is disabled with `--no-default-features` in which case vllm is used.
```

Reply:
Also this answered my above comments on removing `--features mistralrs` from docker builds